src/ui/public/courier/__tests__/saved_object.js

```diff
@@ -88,11 +88,11 @@ describe('Saved Object', function () {
     // The default implementation of safeConfirm uses $timeout which will cause
     // the test environment to hang.
     function ($provide) {
-      $provide.decorator('safeConfirm', () => {
-        return (message) => {
+      $provide.decorator('safeConfirm', () =>
```

Comment:
I think this got really confusing now. The shortest form of what you have here would be:

```js
$provide.decorator('safeConfirm', () => message => window.confirm(message) ? Promise.resolve() : Promise.reject());
```

If you're against the implicit return of a function (and I can see why you might be), something like this would also work:

```js
const resolveConfirm = message => window.confirm(message) ? Promise.resolve() : Promise.reject();
$provide.decorator('safeConfirm', () => resolveConfirm);
```